manifests/init.pp

```diff
@@ -269,7 +269,7 @@
   }
 
   # The whole php configuration directory can be recursively overriden
-  if $php::source_dir {
+  if $php::source_dir != ''{
```

Comment:
in all these conditions I would do something like:

     if $php::source_dir and $php::source_dir != ''{ 

instead of:

     if $php::source_dir != ''{